catroid/src/main/res/layout/brick_set_look_by_index_and_wait.xml

```diff
@@ -1,65 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is deleting this xml intended? I found that it is not possible to load a project that is using this brick.